app/router.js

```diff
@@ -58,6 +58,16 @@ Router.map(function() {
   this.route('school', { path: 'schools/:school_id'});
   this.route('assign-students', {path: '/admin/assignstudents'});
   this.route('myprofile');
+  this.route('course-rollover');
+  this.route('curriculumInventoryReports', {
```

Comment:
Interesting.  I figured if it wasn't in router.js then it didn't exist.  Ok.  I just worry that this will all still be exposed to anyone who checks the URL.  Not for security, but just that this incomplete and possibly buggy feature might get accidentally used.